### DISCUSSION TITLE
Cursor Options Remain Nonfuncftional

### DISCUSSION THREAD

**Discussion by QCloudZoom** [mgth/LittleBigMouse#334] [General]
I just installed the latest release, read through the usage guide, set up the software (adjusted the monitor sizing after measuring), and have noticed that he options for "Adjust Pointer" and Adjust Speed" remain grey out and unusable regardless of the settings I use. I am currently using three different brand televisions as monitors, but as the issue is with mouse settings, I can't imagine that my screens being televisions instead of monitors should make any difference. I DO have custom mouse pointers and have adjusted the speed of the cursor in Windows settings, though. Could that be the issue?

**Comment by SnappyNK** [comment]
Got the same issue, not sure what's wrong.

**Comment by spencerhorspool** [comment]
Did you ever find a fix?
